src/process/automatic/monitor.rs

```diff
@@ -26,7 +28,7 @@ impl OutputMonitor {
 
         on_startup_rx
             .await
-            .context("Monitoring Automatic startup failed")??;
+            .context("Automatic failed on startup")??;
```

Comment:
I only changed message so it indicates a problem with `Automatic` not with its monitoring. Initially I wanted to add message from original Error, but it might be something specific to the host (like full path with username)"